resources/quality/goofoo/pva/goofoo_near_0.40_pva_fine.inst.cfg

```diff
@@ -0,0 +1,24 @@
+[general]
+version = 4
+name = Fine Quality
+definition = goofoo_near
+
+[metadata]
+setting_version = 17
+type = quality
+quality_type = fine
+material = goofoo_pva
+variant = 0.4mm Nozzle
+
+[values]
+wall_thickness = =line_width*2
+adhesion_type = none
+support_pattern = everywhere
+support_pattern = grid
+support_infill_rate = 50
+support_brim_enable = True
+support_interface_enable = True
+support_enable = True
+# retraction_enable = False
+# retraction_amount = 1
+# retraction_speed = 1
```

Comment:
```suggestion
```

Reply:
Thank you for your suggestions! I have modified it! Please check it! Thanks!